subjects/cybersecurity/image-inspector/audit/README.md

```diff
@@ -0,0 +1,96 @@
+#### General
+
+##### Check the Repo Content
+
+Files that must be inside the repository:
+
+- Detailed documentation in the `README.md` file.
+- Source code for the Image Inspector tool.
+- Any required configuration files and scripts for running the tool.
+
+###### Are all the required files present?
+
+##### Play the Role of a Stakeholder
+
+Organize a simulated scenario where the student takes on the role of a Digital Forensics Expert and explains their solution and knowledge to a team or stakeholder. Evaluate their grasp of the concepts and technologies used in the project, their communication efficacy, and their critical thinking about their solution and the knowledge behind this project.
+
+Suggested role play questions include:
+
+- What is metadata in the context of digital images, and why is it important?
+- How does steganography work, and what are its potential uses and risks?
+- What challenges did you face while developing the Image Inspector tool, and how did you address them?
```

Comment:
then they can say there are no faced problems, this question mainly to create conversation between the auditor and the students

Reply:
So how this question can be evaluated, and what if the auditor count it as false this will lead the project to the failing while it shouldn't, can you please list to me the cases where the auditor can count it as wrong answer.

Reply:
this is not the audit question this is just a simple question to create a conversation